### ISSUE TITLE
Show how to enable debug mode

### ISSUE THREAD

**Issue by jean-philippe-martin** [GoogleCloudPlatform/appengine-python-bottle-scaffold#2]
Thank you for the "hello world" scaffold, this is great for getting started.

Another thing that's great for getting started is getting useful debug output when something goes wrong.

I would suggest updating this scaffold so it turns on [debug mode](http://bottlepy.org/docs/stable/api.html#bottle.debug).

**Comment by webmaven** [comment]
Is this still an active issue?

**Comment by jonparrott** [comment]
No, this is no longer an active project. @elibixby can you disable issue tracking on these projects?
